hazelcast-client/src/main/java/com/hazelcast/client/proxy/txn/TransactionContextProxy.java

```diff
@@ -89,6 +89,18 @@ public void rollbackTransaction() {
     }
 
     @Override
+    public void suspendTransaction() {
+        // TODO: implemet
```

Comment:
Do you plan to "implement" the methods in this PR?

Reply:
Initially I thought to it within follow-up tasks, in case of this PR will be approved and merged.